chapters/memory-layout/memory-operations/drills/tasks/pixels/solution/pixel.h

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef __PIXEL_H__
+#define __PIXEL_H__
+
+typedef struct Pixel {
+	unsigned char R;
+	unsigned char G;
+	unsigned char B;
+} Pixel;
+
+typedef struct Picture {
+	int height;
+	int width;
+	Pixel **pix_array;
```

Comment:
```suggestion
struct pixel {
	unsigned char R;
	unsigned char G;
	unsigned char B;
};

struct picture {
	int height;
	int width;
	struct pixel **pix_array;
```
It's bad practice to use `typedef`s for structs. They're generally reserved for aliases such as `size_t` for `unsigned long`.